sdk/src/impl/whirlpool-impl.ts

```diff
@@ -584,15 +584,24 @@ export class WhirlpoolImpl implements Whirlpool {
 
     txBuilder.addInstruction(positionIx);
 
-    const txBuilders: TransactionBuilder[] = [];
-
-    if (!tokenAccountsTxBuilder.isEmpty()) {
-      txBuilders.push(tokenAccountsTxBuilder);
+    if (tokenAccountsTxBuilder.isEmpty()) {
+      return [txBuilder]
     }
 
-    txBuilders.push(txBuilder);
+    // This handles an edge case where the instructions are too
+    // large to fit in a single transaction and we need to split the
+    // instructions into two transactions.
+    if (
+      !tokenAccountsTxBuilder.isEmpty() &&
+      affiliatedMints.hasNativeMint &&
+      resolveAtaIxs.length >= 2 &&
+      this.ctx.opts.accountResolverOptions?.createWrappedSolAccountMethod != "ata"
+    ) {
```

Comment:
Is there reason to avoid using ``checkMergedTransactionSizeIsValid`` to check if 2 transaction builder can be merged within tx size limit ?

https://github.com/orca-so/whirlpools/blob/main/sdk/src/instructions/composites/collect-all-txn.ts#L174-L183

To reduce additional fetch of blockhash, we can use this dummy because it is used as "padding data".
https://github.com/orca-so/orca-sdks/blob/main/packages/common-sdk/src/web3/transactions/constants.ts#L10

Reply:
Nice. Yep measurement blockhash is what I needed since I was hesitant of using because it is async